project-koku/koku
# Cost 1269 azure parquet duplicates bugfix

## Summary
This fixes a bug introduced in the parquet column name post processing. 

## Testing
1. Set envs 
```
ENABLE_PARQUET_PROCESSING=True
S3_BUCKET_NAME=koku-bucket
S3_ENDPOINT=http://kokuminio:9000
S3_ACCESS_KEY=kokuminioaccess
S3_SECRET=kokuminiosecret
```
2. `make create-test-customer`
3. `make load-test-customer-data`
4. Watch logs to see no error for Azure parquet conversion
5. Check Azure APIs after processing completes.
